docs/multi-node.qmd

```diff
@@ -2,11 +2,12 @@
 title: Multi Node
 description: How to use Axolotl on multiple machines
 ---
+# Distributed Finetuning For Multi-Node with Axolotl and FSDP
```

Comment:
Let's also change the `title: ` yaml to reflect this

Reply:
Thanks for the review,
But I couldn't understand with what you what to change, can you please rephrase or mention the exact changes, if possible.

Reply:
I added the suggestion above that can be committed directly.

Reply:
Done, Thanks